src/UWP/ArcGISRuntime.UWP.Samples/ArcGISRuntime.UWP.Samples.VBNet.vbproj

```diff
@@ -109,6 +110,8 @@
     </Content>
     <None Include="Samples\Layers\RasterLayerImageServiceRaster\metadata.json" />
     <None Include="Samples\Layers\RasterLayerImageServiceRaster\README.md" />
+    <None Include="Samples\Layers\RasterLayerRasterFunction\metadata.json" />
+    <None Include="Samples\Layers\RasterLayerRasterFunction\README.md" />
```

Comment:
I don't think we want to include the readme.md files in the solutions/projects. It looks like the readme was included for the RasterLayerImageServiceRaster sample as well, so I don't know how you want to handle that.